packages/dids/src/methods/did-dht.ts

```diff
@@ -1253,19 +1257,29 @@ export class DidDhtDocument {
     }
 
     // Add a DNS TXT record for the root record.
-    dnsAnswerRecords.push({
+    txtRecords.push({
       type : 'TXT',
       name : '_did.' + DidDhtDocument.getUniqueDidSuffix(didDocument.id) + '.', // name of a Root Record MUST end in `<ID>.`
       ttl  : DNS_RECORD_TTL,
       data : rootRecord.join(PROPERTY_SEPARATOR)
     });
 
+    // Add an NS record for each authoritative gateway URI.
+    for (const gatewayUri of authoritativeGatewayUris || []) {
+      nsRecords.push({
+        type : 'NS',
+        name : '_did.' + DidDhtDocument.getUniqueDidSuffix(didDocument.id) + '.', // name of an NS record a authoritative gateway MUST end in `<ID>.`
+        ttl  : DNS_RECORD_TTL,
```

Comment:
this is not configurable in the spec?

Reply:
It's configurable in the sense that it's an optional parameter for this method.